docs/installation.rst

```diff
@@ -14,12 +14,7 @@ Run locally
 For development
 ---------------
 
-By default, *Kinto* persists the records and internal cache in a PostgreSQL
-database.
-
-The default configuration will connect to the ``postgres`` database on
-``localhost:5432``, with user/password ``postgres``/``postgres``.
-See more details below about installation and setup of PostgreSQL.
+By default, *Kinto* persists the records, permissions and internal cache in redis.
```

Comment:
Please remove this changes from this PR (out of scope), it will be adressed in #90 or #95